website/client/src/components/tasks/user.vue

```diff
@@ -483,7 +483,7 @@ export default {
       userTags.forEach(t => {
         if (t.group) {
           tagsByType.groups.tags.push(t);
-        } else if (t.challenge) {
+        } else if (t.challenge && t.challenge === 'true') {
```

Comment:
why are we checking for `tag.challenge` being `"true"`, so the string version of `true` and not the boolean?